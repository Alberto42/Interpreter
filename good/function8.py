def f(x) {
    y = 3
    def f(x) {
        y = 2
    }
    f(42)
    return y
}

y = f(42)