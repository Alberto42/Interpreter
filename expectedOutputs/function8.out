Ident "y" = VInt 2


